data/tutorials/lg_08_error_handling.md

```diff
@@ -30,7 +32,7 @@ instance, in Unix systems, here what is contained in `man 2 read`:
 >
 > On error, -1 is returned, and `errno` is set to indicate the error.
 
-Great software was written using this style. However, since correct are errors
+Great software was written using this style. However, since correct and error
```

Comment:
```suggestion
Great software was written using this style. However, since the expected and error
```

Reply:
Is it "...since the expected error messages can't be distinguished" ?

Or do you mean ... the expected messages including error messages?

Reply:
Maybe something like: “The values which are expected can't be distinguished from the values which represent errors”